### ISSUE TITLE
Calling send on a parameter containing a pointer tensor deletes the data tensor

### ISSUE THREAD

**Issue by robert-wagner** [OpenMined/PySyft#1882]
```
import torch as th
import syft as sy
# sy.create_sandbox(globals(), verbose=False)

hook = sy.TorchHook(th)
bob = sy.VirtualWorker(hook, "bob")
alice = sy.VirtualWorker(hook, "alice")

sy.local_worker.add_worker(bob)

data = th.nn.Parameter(th.tensor([[1.,1],[0,1],[1,0],[0,0]]))

data = data.send(bob).send(alice)
data = data.get().get()
```

**Comment by robert-wagner** [comment]
Calling send on a parameter containing a pointer tensor calls _simplify_pointer_tensor twice
Parameter>[PointerTensor - 67201391700@bob]
simplifying pointer tensor
simplifying pointer tensor
detailing pointing tensor
detailing pointing tensor

**Comment by LaRiffle** [comment]
Solved by #1892 !
